micro-rdk-server/.cargo/config.toml

```diff
@@ -18,9 +18,3 @@ rustflags = ["-C", "default-linker-libraries"]
 linker = "ldproxy"
 runner = "espflash --monitor"
 rustflags = ["-C", "default-linker-libraries"]
-
-
-[env]
-ESP_IDF_TOOLS_INSTALL_DIR = {value = "fromenv", force = true}
-ESP_IDF_VERSION= { value = "v4.4.1", force = true}
```

Comment:
Note that `4.4.1` was inconsistent with what we have in the canon shell, which is 4.4.4